cob_controller_configuration_gazebo/launch/robots/default_controllers_cob4-1.launch

```diff
@@ -36,6 +36,8 @@
 		<arg name="sim" value="true"/>
 	</include>
 
+        <node pkg="rostopic" type="rostopic" args="pub /safety_controller/state_is_valid std_msgs/Bool 'data: true' -r10" name="fake_collission_monitor" output="screen"/>
```

Comment:
- indentation
- why do we need this? Can't we just run the cob_collision_monitor in simulation?